etc/systemd/system/zfs-import-scan.service.in

```diff
@@ -11,7 +11,7 @@ ConditionPathExists=!@sysconfdir@/zfs/zpool.cache
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStart=@sbindir@/zpool import -aN -o cachefile=none
+ExecStart=@sbindir@/zpool import -afN -o cachefile=none
```

Comment:
This seems potentially harmful. Why the change?

Reply:
This change is only for testing, since i update the kernel very often it always fails to import on each reboot since the kernel version has changed. this will be removed later again. its very annoying that zfs fails to import, because the kernel version has just changed